tests/core/test_agent.py

```diff
@@ -83,6 +83,38 @@ def model_server_app(model_path: Text, model_hash: Text = "somehash"):
     ]
 
 
+@pytest.mark.parametrize(
+    "text_message_data, expected",
+    [
+        (
+            '/greet{"name":"Rasa"}',
+            {
+                "text": '/greet{"name":"Rasa"}',
+                "intent": {"name": "greet", "confidence": 1.0},
+                "intent_ranking": [{"name": "greet", "confidence": 1.0}],
+                "entities": [
+                    {"entity": "name", "start": 6, "end": 21, "value": "Rasa"}
+                ],
+            },
+        ),
+        (
+            "text",
```

Comment:
this text should not be passing! I think it should be `/text`

Reply:
In the first example is passed a message using `/`, this **text** is passed to test if this method will handle correctly text messages too. Or this is not needed? I can easily add more tests cases with intent payload without entities too.

Reply:
It should be fine. If you pass a "normal" text (without prefix) to the `RegexInterpreter`, it handles it as if there would be an intent prefix and also adds it to the text. No sure about the use case, but that is the current way of implementation.

Reply:
ah, just saw that now in the code :-D Ok cool 👍